Classes/Public/Manager/Submanagers/OCTSubmanagerCalls.h

```diff
@@ -25,10 +25,13 @@
 
 @interface OCTSubmanagerCalls : NSObject
 
+@property (weak, nonatomic) id<OCTSubmanagerCallDelegate> delegate;
+
 /**
  * Call sessions that are active.
  */
-@property (strong, nonatomic, readonly) OCTArray *calls;
+- (NSSet *)calls;
```

Comment:
User needs to know about call objects updates (to update UI or something). Maybe you'll take a look at OCTBasicContainer and implement something similar to OCTFriendsContainer but for calls?

I'm thinking of creating some generic OCTArray for end user, that will have inside either database objects or OCTBasicContainer. Well, maybe in future I'll do something like that.